pkg/parse/events/funnel_test.go

```diff
@@ -202,6 +204,7 @@ func TestFunnel_Start(t *testing.T) {
 					Event:  Event{Type: RetrySyncEventType}, // 7s
 					Result: Result{},
 				},
+				// No more retry events until another event sets ResetRetryBackoff=true
```

Comment:
We can verify the backoff's steps to validate no more retry.

Reply:
I added another SyncEvent without ResetRetryBackoff to confirm that a RetryEvent won't be called until the ResetRetryBackoff is true.